openmc/material.py

```diff
@@ -311,6 +327,65 @@ def remove_nuclide(self, nuclide):
         if nuclide._name in self._nuclides:
             del self._nuclides[nuclide._name]
 
+    def add_macroscopic(self, macroscopic):
```

Comment:
I'm not sure I understand, do you mean have both a property and an adder method (to match Nuclide and Element?